packages/backend-core/src/cache/docWritethrough.ts

```diff
@@ -0,0 +1,85 @@
+import { AnyDocument, Database } from "@budibase/types"
+
+import { JobQueue, createQueue } from "../queue"
+import * as dbUtils from "../db"
+import { logWarn } from "../logging"
+
+interface ProcessDocMessage {
+  dbName: string
+  docId: string
+  data: Record<string, any>
+}
+
+export const docWritethroughProcessorQueue = createQueue<ProcessDocMessage>(
+  JobQueue.DOC_WRITETHROUGH_QUEUE,
+  {
+    jobOptions: {
+      // We might have plenty of 409, we want to allow running almost infinitely
+      attempts: Number.MAX_SAFE_INTEGER,
```

Comment:
As with the error - I might tune this to be a little lower, then if we reach the max attempts me log warn, that way we have some mechanism to track if things are getting clogged up.

Reply:
We don't know what is a valid max attempts for this. I log the attempt count, we can probably monitor it for a while and then tweak it?

Reply:
My fear with such a high number is just the queue getting overloaded, theres basically no way to clear it down with this, if the queue gets full we can't wait for it to settle, we'll need to go into Redis and empty it out manually. At least if the number of attempts is something say like `100` - eventually it will clear out, even if that involves dumping some events.

Reply:
Ok, I set it to 100 and we can then see if errored messages are piling up